vyper/semantics/validation/utils.py

```diff
@@ -20,7 +20,11 @@
 from vyper.semantics.namespace import get_namespace
 from vyper.semantics.types.abstract import IntegerAbstractType
 from vyper.semantics.types.bases import BaseTypeDefinition
-from vyper.semantics.types.indexable.sequence import ArrayDefinition, TupleDefinition
+from vyper.semantics.types.indexable.sequence import (
+    ArrayDefinition,
```

Comment:
Should this be `StaticArrayDefinition` to match?

Reply:
Could be, I didn't want to change the existing type name

Reply:
Haha https://github.com/vyperlang/vyper/pull/2556/files#diff-19b7b9e98dc724b69e793038c0d43aff56b4b5c9636a37ffef6bff7332a29432R57